custom/logistics/api.py

```diff
@@ -212,3 +215,13 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
             if apply_updates(user, user_dict):
                 user.save()
         return user
+
+    def add_language_to_user(self, logistics_sms_user):
+        domain_part = "%s.commcarehq.org" % self.domain
+        username_part = "%s%d" % (logistics_sms_user.name.strip().replace(' ', '.').lower(),
+                                  logistics_sms_user.id)
+        username = "%s@%s" % (username_part[:(128 - (len(domain_part) + 1))], domain_part)
+        user = CouchUser.get_by_username(username)
+        if user:
```

Comment:
can you change this to `if user and user.language != logistics_sms_user.language` to avoid unnecessary saves?